server/views/pages/collections.html

```diff
@@ -31,11 +31,10 @@ <h1 id="title" class="govuk-heading-xl">{{ title }}</h1>
 {% endblock %}
 
 {% block content %}
-
-  {% if data.relatedContent|length %}
-    {{ hubRelatedContent({
+  {% if data.hubContent|length %}
+    {{ hubContent({
       id: tagId,
-      data: data.relatedContent,
+      data: data.hubContent,
```

Comment:
Component and data having the same name is potentially confusing

Reply:
refactored